static/templates/hotspot_overlay.hbs

```diff
@@ -1,7 +1,7 @@
-<div id="hotspot_{{name}}_overlay" class="hotspot overlay" data-overlay="hotspot_{{name}}_overlay">
-    <div class="hotspot-popover">
+<div role="dialog" aria-labelledby="hotspotTitle" id="hotspot_{{name}}_overlay" class="hotspot overlay" data-overlay="hotspot_{{name}}_overlay">
+     <div class="hotspot-popover">
```

Comment:
The pipeline is failing because if this indentation change. https://github.com/zulip/zulip/pull/16471/checks?check_run_id=1203279041